src/qibolab/qubits.py

```diff
@@ -71,7 +71,7 @@ class Qubit:
     twpa: Optional[Channel] = None
     drive: Optional[Channel] = None
     flux: Optional[Channel] = None
-    flux_coupler: Optional[List["Qubit"]] = None
+    flux_coupler: Optional[List["Qubit"]] = field(default_factory=dict)
```

Comment:
`Optional[List]` means that is either a list or `None`. No chance it could be a `dict`.